### PR TITLE
Update Find feature to include Rating

### PR DESCRIPTION
**Pull request by Ardentsoul**
Find command can now find shows based on ratings.

### CODE REVIEW

File: src/main/java/seedu/trackermon/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all shows whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "At least one parameter must be stated in the find command.\n"
-            + "Parameters: [NAME] [STATUS] [TAG]\n"
+            + "Parameters: [NAME] [STATUS] [RATING] [TAG]...\n"
```
Comment (aweijun) [inline src/main/java/seedu/trackermon/logic/commands/FindCommand.java:22]: Should it be [t/TAG]… instead?

File: src/main/java/seedu/trackermon/model/show/RatingContainsKeywordsPredicate.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package seedu.trackermon.model.show;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.trackermon.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Show}'s {@code Name} matches any of the keywords given.
```
Comment (aweijun) [inline src/main/java/seedu/trackermon/model/show/RatingContainsKeywordsPredicate.java:9]: Just clarifying shouldn't this be Rating rather than Name?

**Review by aweijun — APPROVED** [review]
Other than a few nitpick, everything else LGTM.

File: src/main/java/seedu/trackermon/model/show/RatingContainsKeywordsPredicate.java (hunk 2 above)
Comment (Ardentsoul) [reply src/main/java/seedu/trackermon/model/show/RatingContainsKeywordsPredicate.java:9]: Yes, good spot!

File: src/main/java/seedu/trackermon/logic/commands/FindCommand.java (hunk 1 above)
Comment (Ardentsoul) [reply src/main/java/seedu/trackermon/logic/commands/FindCommand.java:22]: Ah yes, I will add prefixes to all of them.

**Review by arcornior — APPROVED** [review]
LGTM!

**Review by JonathanHoshi — APPROVED** [review]
LGTM!

**Pull request merged**